doc/manual/noteworthy-differences.rst

```diff
@@ -6,6 +6,44 @@
 Noteworthy Differences from other Languages
 *******************************************
 
+Noteworthy unusual Julia features
+---------------------------------
+
+* Julia allows identifiers with complex unicode, for example you could represent the value of a chi-squared test with ``χ²``, ``χ⁽³⁾`` is used in electromagnetism for the third-order nonlinear susceptibility and ``∇²`` is used in electromagnetism for the third-order nonlinear susceptibility. All of these unicode symbols are valid Julia identifiers.
```

Comment:
You didn't change the description of `∇²`, which is often used to denote the Laplacian.